tests/verify-present-mount.yml

```diff
@@ -0,0 +1,39 @@
+---
+#
+# Verify that device is present in the /etc/fstab file
+#
+- name: Read /etc/fstab file
+  shell: cat /etc/fstab
+  register: fstab_buf
+
+- name: Verify that device is present in /etc/fstab file
+  assert:
+    that: fstab_buf.stdout.find(device_path) != -1
+    msg: "{{ device_path }} is not present in /etc/fstab"
+
+#
+# Verify that the device has the correct mount location
+#
```

Comment:
Most, if not all, of the rest should use `ansible_mounts` since the information is already in the setup facts.